olive/workflows/run/config.py

```diff
@@ -246,13 +243,6 @@ def validate_engine(cls, v, values):
         v = _resolve_system(v, values, "target")
         return _resolve_evaluator(v, values)
 
-    @validator("engine")
```

Comment:
is this removed since the engine does the check anyways and the info log is too verbose?

Reply:
Yeah 2 reasons:
- Engine does the check.
- This log level is not controlled by logging.py as this runs before we are initializing logging.py